notebooks/225-distilbert-sequence-classification/README.md

```diff
@@ -0,0 +1,10 @@
+# Sequence Classification OpenVINO API 2.0
+This notebook performs sequence classification using OpenVINO API 2.0. You can alternatively replace the model with the other bert-based models for sequence classification.
+## Notebook Contents
+We'll be using distilbert-base-uncased-finetuned-sst-2-english transformers based model from Huggingface. Details of the model can be found [here].(https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english)
+![image](https://user-images.githubusercontent.com/95271966/206130638-d9847414-357a-4c79-9ca7-76f4ae5a6d7f.png)
+
+
+## Installation Instructions
+

```

Comment:
not sure that it is relevant information for notebook.
Please align description with other notebooks
https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/001-hello-world#installation-instructions

Reply:
Done